apis/r/R/Init.R

```diff
@@ -25,10 +25,6 @@
     } else {
       .pkgenv[["use_current_domain_transitional_internal_only"]] <- FALSE
     }
-    if (rpkg_lib_version != soma_lib_version) {
```

Comment:
Appears to be unrelated to this PR -- should be a separate PR if I understand correctly.

Reply:
How come?  `git blame` pins it at five months ago

```diff
5 months ago    Dirk Eddelbuet  0a6ca6b68│ 15 │     if (rpkg_lib_version != soma_lib_version) {
                                         │ 16 │         msg <- sprintf("TileDB Core version %s used by TileDB-R package, but TileDB-SOMA uses %s",
                                         │ 17 │                        sQuote(rpkg_lib_version), sQuote(soma_lib_version))
                                         │ 18 │         stop(msg, call. = FALSE)
                                         │ 19 │     }
```

Reply:
If you expand the line you see that

- this is copy-paste from a few lines above
- it uses a variable that is not defined (which `R CMD check` flagged and how I got there)
- assigns to a variable that is not used

So can we maybe turn it around and let this PR remove the non-working code and re-inject in a future PR what you intended, fully fleshed=out?

Reply:
For reference:

https://github.com/single-cell-data/TileDB-SOMA/blob/b3feada0d1282442d1190f56a43670a32dad97ae/apis/r/R/Init.R#L28-L31

where `cdmsg` is a free variable never assigned, and `msg` is a temporary not used.

Reply:
Thanks for the clarify @eddelbuettel !